scaleway/resource_apple_silicon_server.go

```diff
@@ -48,7 +50,24 @@ func resourceScalewayAppleSiliconServer() *schema.Resource {
 				Description: "VNC url use to connect remotely to the desktop GUI",
 				Computed:    true,
 			},
-
+			"state": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     applesilicon.ServerStatusStarting.String(),
```

Comment:
Should it be a computed state?

Reply:
In fact, it must be only computed to be read only